include/player.hpp

```diff
@@ -200,6 +200,8 @@ class Player {
 
   float GetAudioBalance() { return Internal::PlayerGetAudioBalance(id_); }
 
+  float GetBufferingPosition() { return Internal::PlayerGetBufferingProgress(id_); }
```

Comment:
Should it be `GetBufferingProgress` or `GetBufferingPosition`? Most likely, as in case of `DART_VM`, you'll need event streams, so an event listener would be preferrable too. Not sure if this PR is in draft state as of now.